core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -864,6 +869,20 @@ public VirtualHostBuilder accessLogWriter(AccessLogWriter accessLogWriter, boole
         return this;
     }
 
+    /**
+     * Sets the {@link ScheduledExecutorService} dedicated to the execution of blocking tasks or invocations.
+     * If not set, {@linkplain CommonPools#blockingTaskExecutor() the common pool} is used.
+     *
+     * @param shutdownOnStop whether to shut down the {@link ScheduledExecutorService} when the
+     *                       {@link Server} stops
+     */
+    public VirtualHostBuilder blockingTaskExecutor(ScheduledExecutorService blockingTaskExecutor,
+                                                   boolean shutdownOnStop) {
```

Comment:
Ditto. Should we need `blockingTaskExecutor(numThreads)` for users' convenience?